nomenclature/processor/region.py

```diff
@@ -408,7 +408,8 @@ def apply(self, df: IamDataFrame, dsd: DataStructureDefinition) -> IamDataFrame:
 
         if not processed_dfs:
             raise ValueError(
-                f"The region aggregation for model {model} resulted in an empty dataset"
+                f"The region aggregation for model(s) {df.model} resulted in an empty "
+                "dataset."
```

Comment:
For consistency with the class **RegionProcessor** (if you agree, please also change in other parts of the logging)...

```suggestion
                f"The region-processing for model(s) {df.model} returned an empty dataset"
```